assets/js/components/Project/Form.jsx

```diff
@@ -58,12 +87,15 @@ const ProjectForm = ({ showForm, setShowForm }) => {
 
                 <UIFormField label="Project Title">
                   <UIFormInput
+                    id={project.id || formType}
```

Comment:
Seems like we're using `id` for something other than the attribute is intended by putting `formType` value on it.  Is there another way to communicate this `formType`?

Reply:
I tried with key, key only updates the value part. if the id is not unique, since we use the same form twice on a single page /projects to add and edit, the id is coming out as same causing this error 
<img width="1183" alt="Screen Shot 2021-01-20 at 9 51 29 AM" src="https://user-images.githubusercontent.com/72217877/105200551-93594f80-5b05-11eb-944c-79bfbd73a231.png">

Reply:
I can do ` id={`${formType}${project.id}`}` to incorporate both and not make it awkward.